### PR TITLE
add description for raspberry pi switch

### PR DESCRIPTION
**Pull request by niccokunzmann**
I added our Raspberry Pi switch to this so that other people can build on top of it. Maybe they will improve it and I can make mine better :)
There is a documentation about how to fork and setup. It is quite easy.

### CODE REVIEW

**Comment by niccokunzmann** [comment]
Hm. How would I put that here? https://spaceapi.io/provide-an-endpoint/#stuff-you-can-use

**Comment by dbrgn** [comment]
> Hm. How would I put that here? https://spaceapi.io/provide-an-endpoint/#stuff-you-can-use

Yeah, it probably makes sense to put it into the `operation` category, not `space-visualization`!

File: content/stuff/raspberry-pi-button/contents.lr [hunk 1]
```diff
@@ -0,0 +1,19 @@
+_model: stuff
+---
+title: Raspberry Pi Status Switch
+---
+author: Nicco Kunzmann
+---
+category: space-visualization
+---
+description: This provides source code and a directory endpoint for the hackerspace status baes on a Raspberry Pi attached to a button. You can switch the space on and off.
```
Comment (dbrgn) [inline content/stuff/raspberry-pi-button/contents.lr:9]:
```suggestion
description: This provides source code and a directory endpoint for the hackerspace status based on a Raspberry Pi attached to a button. You can switch the opening status on and off.
```

**Review by dbrgn — CHANGES_REQUESTED** [review]

File: content/stuff/raspberry-pi-button/contents.lr [hunk 2]
```diff
@@ -0,0 +1,19 @@
+_model: stuff
+---
+title: Raspberry Pi Status Switch
+---
+author: Nicco Kunzmann
+---
+category: space-visualization
```
Comment (dbrgn) [inline content/stuff/raspberry-pi-button/contents.lr:7]:
```suggestion
category: operation
```

**Comment by niccokunzmann** [comment]
@dbrgn, I merged the suggestions. Thanks!

**Pull request merged**